packages/components/Atoms/Icon/icons/facebook.js

```diff
@@ -2,11 +2,11 @@ import Icon from '../Icon.vue'
 
 Icon.register({
   'facebook': {
-    width: 8,
-    height: 16,
+    width: 32,
+    height: 32,
```

Comment:
@dylankelly the Facebook icon was changed in size as requested in the ticket. This was the new size provided in an attempt to get all social media icons to line up using consistent sizing. 

See the same change made to the email_solid icon

Reply:
Sure, but then the component usage is broken See https://www.chromatic.com/test?appId=5e736ff82649250022dd830c&id=61ea3cdca321cc003ab63c88. I think this PR should fix the component usage too, or there should be a follow up ticket at least.

Reply:
Understood. That's exactly why I was asking for some advice and a review. I suspected this would have wider implications but didn't have a clue what those implications would be and where. 

Can you help us understand, what is the correct course of action / way forward here?

Reply:
requested 16x16 SVG icons -- will see how that goes